odoo/models.py

```diff
@@ -2965,7 +2965,14 @@ def unlink(self):
             Defaults = self.env['ir.default'].sudo()
             Attachment = self.env['ir.attachment']
 
-            for sub_ids in cr.split_for_in_conditions(self.ids):
+            # Removing virtual ids. In some cases,
+            # virtual ids are passed to the backend
+            # and we don't handle them
+            ids = [i for i in self.ids if 'virtual' not in str(i)]
+            if not ids:
+                pass
```

Comment:
?

Reply:
Thanks, I forgot to replace it while debugging